EmotiBitFirmwareInstaller/src/ofApp.cpp

```diff
@@ -662,13 +670,15 @@ ofApp::Board ofApp::getBoardFromDeviceInfo(ofx::IO::SerialDeviceInfo deviceInfo)
 		if (info.desc.find(espDescIdentifier) != std::string::npos)
 		{
 			// It is ESP!
+			ofLogNotice("Board Detected") << "Feather ESP";
 			return Board::FEATHER_ESP_32;
 		}
 #elif defined TARGET_OSX
-        // if descriptions says silicon labs and port is SLAB_USBtoUART
-        if (info.desc.find(espDescIdentifier) != std::string::npos && info.port.find(espSlabIdentifier) != std::string::npos)
+        // if port is SLAB_USBtoUART
```

Comment:
This is the main change in the code. 
Everything else is primarily a logging activity.